builds/jenkins-build-pack-api/Jenkinsfile

```diff
@@ -259,6 +259,8 @@ node() {
 						sh "aws apigateway put-rest-api --rest-api-id ${aws_api_id} --mode merge --parameters basepath=prepend --body 'file://swagger/swagger.json'" + " --profile cloud-api"
 						sh "aws apigateway create-deployment --rest-api-id ${aws_api_id} --stage-name ${current_environment} --profile cloud-api"
 						sh "aws apigateway tag-resource --resource-arn arn:aws:apigateway:${configLoader.AWS.REGION}::/restapis/${aws_api_id}/stages/${current_environment} --tags Application=Jazz,JazzInstance=${configLoader.INSTANCE_PREFIX}  --profile cloud-api"
+						sh "aws apigateway update-stage --rest-api-id ${aws_api_id} --stage-name ${current_environment} --patch-operations op=replace,path=/*/*/logging/loglevel,value=INFO --region ${configLoader.AWS.REGION}"
```

Comment:
Hmm, this will enable logs for the endpoint with each and every build. Ideally this should be done during resource provisioning (API Gateway endpoint).

Let's remove it from here and fix this @ installer side.

Reply:
Removed.